app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/address/enter_postcode.scala.html

```diff
@@ -29,33 +29,34 @@
 @import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.RepresentativeType
 
 @this(
-mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
-pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading,
-textInput: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.text_input,
-submitButton: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.submit_button,
-errorSummary: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.error_summary,
-formWrapper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
-backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link,
-accountMenu: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.account_menu,
-returnToSummaryLink: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.return_to_summary_link,
-caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption
+    mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
+    pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading,
+    textInput: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.text_input,
+    submitButton: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.submit_button,
+    errorSummary: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.error_summary,
+    formWrapper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
+    backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link,
+    accountMenu: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.account_menu,
+    returnToSummaryLink: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.return_to_summary_link,
+    caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption
 )
 
-@(form: Form[AddressLookupRequest],
-        backLink: Call,
-        submit: Call,
-        enterUkAddress: Call,
-        addressJourneyType: AddressJourneyType,
-        isATrust: Boolean,
-        representativeType: Option[RepresentativeType]
+@(
+    form: Form[AddressLookupRequest],
+    backLink: Call,
+    submit: Call,
+    enterUkAddress: Call,
+    addressJourneyType: AddressJourneyType,
+    isATrust: Boolean,
+    representativeType: Option[RepresentativeType]
 )(implicit request: RequestWithSessionData[_], messages: Messages, appConfig: ViewConfig)
 
 @key = @{"postcode"}
 @isAgent = @{ request.userType.contains(UserType.Agent) }
 @userKey = @{
   representativeType match {
       case Some(PersonalRepresentative) => ".personalRep"
-      case Some(PersonalRepresentativeInPeriodOfAdmin) => ".personalRep"
+      case Some(PersonalRepresentativeInPeriodOfAdmin) => ".personalRepInPeriodOfAdmin"
```

Comment:
shouldn't this be something like this?
```scala
@userKey = @{
  representativeType match {
      case Some(PersonalRepresentative) => ".personalRep"
      case Some(PersonalRepresentativeInPeriodOfAdmin)  => if(isAgent) ".agent" else ".personalRep"
      case Some(Capacitor) => ".capacitor"
      case None =>
          if (isAgent) ".agent"
          else if (isATrust) ".trust"
          else ""
  }
}
```
the content in the ticket looks like it's specifically for _agents_ of period of admin people.

Reply:
we dont need separate entry for ".personalRepInPeriodOfAdmin" in messages file?

Reply:
cos there's no specific message for non-agent period of admin users I assume that it would follow the same content as personal representatives. We could either reuse the same key for periodOfAdmin or create a new one and put the same content for personalRep in it. We may end up with a lot of duplicate messages though with the latter option.